app/assets/stylesheets/admin/admin.sass

```diff
@@ -0,0 +1,47 @@
+// -----------------------------------------------------------------------------
+// This file contains all Admin styles.
+// -----------------------------------------------------------------------------
+
+@import "normalize-rails"
+@import "bourbon"
+
+/*!
+ * Author:       The Frontier Group
+ * Author URI:   http://www.thefrontiergroup.com.au/
+ */
```

Comment:
Is there any reason we have two comment blocks? Can they be combined in some way?